rotkehlchen/db/updates.py

```diff
@@ -81,6 +84,64 @@ def _get_remote_info_json(self) -> dict[str, Any]:
 
         return json_data
 
+    def update_rpc_nodes(self, local_version: int, to_version: int) -> None:
+        """
+        Check the updates in the inclusive range [local_version + 1, to_version] and update the
+        rpc nodes for those versions.
+        """
+        for version in range(local_version + 1, to_version + 1):
+            file_url = RPC_NODES_URL.format(branch=self.branch, version=version)
+            try:
+                updates = query_file(file_url, is_json=True)
+            except RemoteError as e:
+                log.warning(f'Failed to update rpc nodes due to {str(e)}')
+                break
+
+            updated_rpc_nodes = updates.get('rpc_nodes')
+            if updated_rpc_nodes is None:
+                log.error(f'Remote update {file_url} does not contain rpc_nodes key')
+                break
```

Comment:
All these, from line 92 to 104 is common between this and `update_spam_assets`. Abstract it away please.